content/docs/examples/advanced-egress/egress-gateway-mtls-origination/index.md

```diff
@@ -0,0 +1,563 @@
+    EOF
+    {{< /text >}}
+
+1.  Define an environment variable to hold the name of the `sleep` pod:
+
+    {{< text bash >}}
+    $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})
+    {{< /text >}}
+
+1.  Use the deployed [sleep]({{< github_tree >}}/samples/sleep) container to send requests to the NGINX server. Since
+    the host `nginx.example.com` does not exist and will not be resolved by DNS, use the `--resolve` option of `curl` to
+    resolve the hostname manually. You can provide any IP to the `--resolve` option, except for `127.0.0.1`, for example
+    `1.1.1.1`.
+    Istio will route the request correctly to your NGINX server. In the real life scenarios, a DNS entry will exist for
+    the destination hostname and providing `--resolve` option to `curl` will not be required.
+
+    {{< text bash >}}
+    $ kubectl exec -it $SOURCE_POD -c sleep -- curl -v --resolve nginx.example.com:443:1.1.1.1 --cacert /etc/nginx-ca-certs/ca-chain.cert.pem --cert /etc/nginx-client-certs/tls.crt --key /etc/nginx-client-certs/tls.key https://nginx.example.com
+    ...
+    Server certificate:
+      subject: C=US; ST=Denial; L=Springfield; O=Dis; CN=nginx.example.com
+      start date: 2018-08-16 04:31:20 GMT
+      expire date: 2019-08-26 04:31:20 GMT
+      common name: nginx.example.com (matched)
+      issuer: C=US; ST=Denial; O=Dis; CN=nginx.example.com
+      SSL certificate verify ok.
+    > GET / HTTP/1.1
+    > User-Agent: curl/7.35.0
+    > Host: nginx.example.com
+    ...
+    < HTTP/1.1 200 OK
+
+    < Server: nginx/1.15.2
+    ...
+    <!DOCTYPE html>
+    <html>
+    <head>
+    <title>Welcome to nginx!</title>
+    ...
+    {{< /text >}}
+
+1.  Verify that the server requires the client's certificate:
+
+    {{< text bash >}}
+    $ kubectl exec -it $(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name}) -c sleep -- curl -k --resolve nginx.example.com:443:1.1.1.1 https://nginx.example.com
+    <html>
+    <head><title>400 No required SSL certificate was sent</title></head>
+    <body bgcolor="white">
+    <center><h1>400 Bad Request</h1></center>
+    <center>No required SSL certificate was sent</center>
+    <hr><center>nginx/1.15.2</center>
+    </body>
+    </html>
+    {{< /text >}}
+
+## Redeploy the Egress Gateway with the client certificates
+
+1. Create Kubernetes [Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) to hold the client's and CA
+   certificates.
+
+    {{< text bash >}}
+    $ kubectl create -n istio-system secret tls nginx-client-certs --key nginx.example.com/4_client/private/nginx.example.com.key.pem --cert nginx.example.com/4_client/certs/nginx.example.com.cert.pem
+    $ kubectl create -n istio-system secret generic nginx-ca-certs --from-file=nginx.example.com/2_intermediate/certs/ca-chain.cert.pem
+    {{< /text >}}
+
+1.  Generate the `istio-egressgateway` deployment with a volume to be mounted from the new secrets. Use the same options
```

Comment:
Please be specific. In the command the `istio-egressgateway` text can be found multiple times but you are referring to the object after `--name`. A better description of the command would be to configure the deployment of the egress gateway, is it not? I would also refrain from referencing the "options you used for generating your `istio.yaml`. You provide all the options in the command and the sentence would block readers forcing them to try to go back and find those options only to realize you had them in the command all along. I suggest:
1.  Configure the deployment of the egress gateway with the following command. The command includes all the options the `istio-egressgateway` pod requires to mount a volume from the new secrets.

Reply:
"The sentence would block readers forcing them to try to go back and find those options only to realize you had them in the command all along"

Nope, I do not have the additional options the users used. See https://istio.io/docs/setup/kubernetes/helm-install/#customization-example-traffic-management-minimal-set:

> Installation Options has the full list of options allowing you to tailor the Istio installation to your needs.

The full list of options is here - https://istio.io/docs/reference/config/installation-options/. The point in the text is: Use the same options you used to install Istio, plus the options in this command.

Reply:
> A better description of the command would be to configure the deployment of the egress gateway, is it not?

This step generates a spec of `istio-egressgateway` `Deployment`, `Deployment` is a kind of configuration items in Kubernetes, the readers know it since it is an advanced example. Probably the word "spec" is missing here, or maybe the "deployment" should be `Deployment`. 

The next step _deploys_ the spec of the `Deployment`, or in `kubectl` terms _applies_ or _creates_ a `Deployment`.